src/context/IntlContext.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { IntlProvider } from "react-intl";
+import App from "../App";
+import Spanish from "../lang/es-US.json";
+import English from "../lang/en-US.json";
+
+export const Context = React.createContext();
+
+// detect the language preference of the user's browser
+export const locale =
+  window.navigator.userLanguage || window.navigator.language;
+
+let messages;
+if (locale === "en-US") {
+  messages = English;
+} else {
+  messages = Spanish;
+}
+
+const IntlContext = ({ children }) => (
+  <Context.Provider value={{ locale }}>
+    <IntlProvider locale={locale} messages={messages}>
+      <App />
+    </IntlProvider>
+    {children}
```

Comment:
🔧 `{children}` should be where `<App />` is. `<App />` will be the child component passed in from `main.jsx`.

Reply:
Addressed:  082ccb0f91519e5b421b33e7e7453b542a2330c3